schema.sql

```diff
@@ -29,4 +29,11 @@ CREATE TABLE visits( vet_id INT ,
                     date_of_visit DATE,
                     PRIMARY KEY(vet_id,animal_id),
                     CONSTRAINT fk_vet FOREIGN KEY(vet_id) REFERENCES vets(id), 
-                    CONSTRAINT  fk_animals FOREIGN KEY(animal_id) REFERENCES animals(id));
+                    CONSTRAINT  fk_animals FOREIGN KEY(animal_id) REFERENCES animals(id))
+                    ;
+                    
+                    
+/--------------------------------------/
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+CREATE INDEX visits_vets_id_asc ON visits(vets_id ASC);
```

Comment:
- [ ] <b>OPTIONAL:</b> You are missing two more indexes to make it perfect. adding indexes for FOREIGN KEYS or on columns frequently used in queries is one of the basic solutions for the performance problems. I  really recommend you to try it out and see how it would improve what you currently have.

```sql
CREATE INDEX ON visits (animal_id);
CREATE INDEX ON owners (email);